docker/drama-free-django/test.sh

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/env bash
 
-docker run -v `pwd`:/cfgov centos:6 /cfgov/docker/drama-free-django/_test.sh
+set -e
+
+docker build -t cfgov-dfd-builder docker/drama-free-django
+
+docker run \
+  --rm \
+  -u $(id -u):$(id -g) \
+  -v $(pwd):/cfgov:cached \
```

Comment:
Why is `:cached` used here in the test script but not in the build script? Is this Mac-specific, as documented [here](https://docs.docker.com/docker-for-mac/osxfs-caching/#cached) -- is it a no-op when run on Linux systems?

Reply:
Unfortunately, it fails currently on Linux.  I'll remove it.